docs/api/general_rest_usage.md

```diff
@@ -359,45 +397,35 @@ Authorization: Basic QWxhZGRpbjpvcGVuIHNlc2FtZQ==
 
 ### Error handling
 
-Error handling in the REST API is fully based on HTTP error codes. As a web developer, you are probably familiar with the most common ones: `401 Unauthorized`, `404 Not Found` or `500 Internal Server Error`. The REST API uses those, along with a few more, to allow proper error handling.
+Error handling in the REST API is fully based on the HTTP error codes.
+The most common are: `401 Unauthorized`, `404 Not Found`, or `500 Internal Server Error`.
+The REST API uses them along with a few more, to allow proper error handling.
 
-The complete list of error codes used and the conditions in which they apply are specified in the [reference documentation](https://github.com/ezsystems/ezpublish-kernel/blob/master/doc/specifications/rest/REST-API-V2.rst).
+For the complete list of error codes and the conditions in which they apply, see the [reference documentation](https://github.com/ezsystems/ezpublish-kernel/blob/master/doc/specifications/rest/REST-API-V2.rst).
 
 ### General error codes
 
-A few error codes apply to most resources (if they *are* applicable)
-
-#### 500 Internal Server Error
-
-The server encountered an unexpected condition, usually an exception, which prevented it from fulfilling the request: database down, permissions or configuration error.
-
-#### 501 Not Implemented
-
-Returned when the requested method has not yet been implemented. For eZ Platform, most of Users, User groups, Content items, Locations and Content Types have been implemented. Some of their methods, as well as other features, may return a 501.
-
-#### 404 Not Found
-
-Returned when the request failed because the request object was not found.
-
-#### 405 Method Not Allowed
-
-Returned when the requested REST API resource doesn't support the HTTP verb that was used.
-
-#### 406 Not Acceptable
+A few error codes apply to most resources (if they *are* applicable):
 
-Returned when an accept header sent with the requested isn't supported.
+|Error code|Description|
```

Comment:
Error code is just a number e.g. `404`. Then it has error message e.g. `Not Found`
```suggestion
|Error code|Error message|Description|
```